src/brain_teasers/python/subarray_sum/README.md

```diff
@@ -0,0 +1,32 @@
+# Sum of subarray
```

Comment:
this README file is good, just needs to be more readable by adding Python after the backticks

```python
_your code_
```
you can also add the command line that is used to test your implementation and a conclusion at the end

Reply:
Thanks, I made below changes in three different commit so it will be easy to check:

- added the word "python" after the backticks
- added the instruction for testing the code implementation
- added a summary at the end